src/test/java/de/zalando/typemapper/postgres/PgSerializerTest.java

```diff
@@ -61,8 +58,8 @@ public PgSerializerTest(final Object objectToSerialize, final Object expected) {
     public static Collection<Object[]> generateData() throws SQLException {
         return Arrays.asList(
                 new Object[][] {
-                    {new Date(112, 11, 1, 6, 6, 6), Pattern.compile("2012-12-01 06:06:06.000000 [+-]?\\d{2}:\\d{2}:00")},
-                    {new Date(112, 9, 1, 6, 6, 6), Pattern.compile("2012-10-01 06:06:06.000000 [+-]?\\d{2}:\\d{2}:00")},
```

Comment:
Yes. They have changed `toString` method. Millis will be added if they really exists. See commit [1e1f3c4ab1fbbc5b071b50991da6c34afe92e6f0](https://github.com/pgjdbc/pgjdbc/commit/1e1f3c4ab1fbbc5b071b50991da6c34afe92e6f0)